templates/fort.3.mother

```diff
@@ -1,16 +1,54 @@
+GEOME-STRENG TITLE:%Runnam
```

Comment:
I would call the file `fort.3` forgetting about the `.mother` suffix

Reply:
If we call the mother file 'fort.3', I'm afraid this will lead to errors when generating the final 'fort.3' for sixtrack. We need to concatenate 'fort.3.aux', 'fort.3.mad', and 'mother file ' into 'fort.3'. If the mother file is also named 'fort.3', when we open 'fort.3', we will open the mother file...

Reply:
ok it depends on how you do the operations.
if the `fort.3.mother` in the templates dir is kept in memory and all manipulations are done on the memory instead of the disk, this risk is no longer there...

Reply:
yes, you are right. I just looked through the workflow of generating 'fort.3', we can avoid the risk. I will make the change~~